### ISSUE TITLE
Initial Implementation

### ISSUE THREAD

**Issue by trickeydan** [srobo/kchd#3]
- [x] LED Controller Class
- [x] System Status Controller
- [x] User Request Controller
- [x] Astmetad Controller
- [x] Astprocd Controller
- [ ] Astwifi Controller - Blocked by https://github.com/srobo/astoria/issues/167

**Comment by trickeydan** [comment]
Now closing in favour of other issues.
